projects/canopy/src/lib/table/table.notes.ts

```diff
@@ -71,6 +71,14 @@ and in your HTML:
 
 ## Inputs
 
+### LgTableComponent
+A component that acts as a standard table.
+
+| Name | Description | Type | Default | Required |
+|------|-------------|:----:|:-----:|:-----:|
+| \`\`fullSizeBp\`\` | Sets the minimum screen width from which the full size table layout is displayed. Accepts \`sm\`, \`md\` or \`lg\` | string | 'md' | No |
```

Comment:
I'm not sure about the name of the property, I think it's the '**Bp**' that gets me as it might not be that clear what it stands for at first glance. I think this is only the second time we have used a property with a media breakpoint in it to define component behaviour (the radio stack being the other one), so we don't really have a convention yet.

Annoyingly I'm not sure of an alternative, from the design side the naming conventions are 'wide' and 'narrow', but I don't know if that's useful. 

`showWide="md"`
`fullSizeBreakpoint ="md"`
`wideAt="md"`
`wideBreakpoint="md"`

'wide' instead of `fullSize` is slightly more terse especially when used in the css, and it aligns with designs, but is it descriptive enough. Open to another opinion on that I think.

Reply:
@thatguynamedandy I'm wary of 'wide' since it implies the width of the table rather than the layout used. But, I'm having a hard time coming up with better names. I prefer `fullSizeAt` or `fullSizeBreakpoint` but they still don't seem great.

Reply:
@thatguynamedandy @aaronBery How about `scaleUpAt="md"`?